main.c

```diff
@@ -14,8 +14,13 @@
   #include <SDL/SDL_mixer.h>
 #else
   #include "SDL_ttf.h"
+	#if defined(NXDK)
+  #define DISABLE_AUDIO 1
+	#else
+  #define DISABLE_AUDIO 0
   #include <SDL2/SDL_mixer.h>
```

Comment:
The conditional includes of SDL_mixer should happen like any other conditional compilation of SDL_mixer. Such conditional blocks / scopes should not define anything else. They will make it very hard to enable / disable the feature.